passwordValidation.js

```diff
@@ -6,11 +6,33 @@ export const forbiddenPasswords = ["amG84h6yeQ", "mc9Q20pdjH", "jnT6Q2f8U5"];
  * @param {string} password
  * @returns {boolean}
  */
+
 export default function isValidPassword(password = "") {
   // The following line ensures, that password is always a string, like the number 128 -> string "128"
   if (typeof password !== "string") password = String(password);
 
-  // * * * YOUR CODE GOES IN HERE ... * * *
+  // check if a password is included in the forbiden passwords
+  if (forbiddenPasswords.includes(password)) return false;
+
+  // check if valid password length is 10
+  if (!/^[a-zA-Z0-9]{10}$/.test(password)) return false;
+
+  // check if password doesn't contain special characters has only mixture of upper case, lower case and number
+  if (
```

Comment:
This condition checks if the password contains at least one (upper & lower case letter and number) but if the password contains a special character it will still pass.

-refactor the comment that you provided.

Reply:
Thank you, let me work on it